templates/html_report/css/style.css

```diff
@@ -340,7 +345,33 @@ a[target="_blank"]::before {
     color: #333;
     text-align: left;
 }
-
+.bloc .number, .bloc .number-alternate {
```

Comment:
Some colors definitions are not consistant between uppercase and lowercase (#ABCDEF vs #abcdef).

You should be consistant in your whole project, whatever you decide which case you want to apply, for readability.

Reply:
I suggest to update CSS later, in another PR